src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutShippingContext.php

```diff
@@ -41,21 +42,25 @@ public function __construct(
     }
 
     /**
-     * @Given I have proceeded selecting :shippingMethodName shipping method
-     * @When I proceed with :shippingMethodName shipping method
+     * @Given I have proceeded selecting :shippingMethod shipping method
```

Comment:
What is the reason you changed the implementation of steps in ui context?

Reply:
at one of the API steps, I needed to get `shippingMethod` instead `shipping method name` that implicated changes also in UI steps

Reply:
I understand, but this shouldn't force changes in UI context, you can have it implemented `@Given I have proceeded selecting :shippingMethodName shipping method` in UI context and `@Given I have proceeded selecting :shippingMethod shipping method` in API context